Jenkinsfile.sdk

```diff
@@ -45,7 +45,30 @@ pipeline {
 	}
 	post {
 		always {
-			sh 'rm -rf */node_modules'
+			script {
+				sh '''
+					rm -rf .nyc_output; mkdir -p .nyc_output
+					rm -rf coverage; mkdir -p coverage
```

Comment:
It's very trivial thing but can we rename the folders we create on CI with a `ci_` prefix. Because at first glance it feels like we are deleting the folder that was created by test runners then creating the same folder :D Also the line below `cp -r ./commander/.nyc_output/* ./.nyc_output` looks weird at first.

So maybe just `.ci_nyc_output` and `ci_coverage` are more explicit, then we can know we are manually creating these folders to create reports. Would be great if we could rename the other folders we create with jenkins as well.